lib/croods/controller/member.rb

```diff
@@ -23,14 +23,16 @@ def identifier
         params[resource.identifier]
       end
 
-      def member_params
+      def member_params # rubocop:disable Metrics/AbcSize
         params
           .permit(resource.request_attributes.keys)
           .merge(
             params
               .require(resource.resource_name)
               .permit!
           )
+      rescue StandardError
+        params.permit(resource.request_attributes.keys)
```

Comment:
We need a solution that prevents the error from happening instead of rescuing. Please give instructions on how to reproduce the issue and let's find another way to solve the problem in the current project that needs this.

Reply:
I created the Issue: https://github.com/SeasonedSoftware/croods-rails/issues/55.